### PR TITLE
Update repo folder path dynamically

### PR DESCRIPTION
**Pull request by padmashree9110**
Updating repo folder path based on Distro Version

Closes: #265

### CODE REVIEW

**Comment by anselmolsm** [comment]
Please see https://github.com/intel-aero/meta-intel-aero/pull/265#issuecomment-324411949

**Comment by padmashree9110** [comment]
@anselmolsm patch is updated, Please do verify.

File: recipes-core/images/intel-aero-image.bb [hunk 1]
```diff
@@ -100,6 +100,9 @@ IMAGE_INSTALL += "aero-bios"
 
 enable_repo() {
 		sed -i 's/enabled=0/enabled=1/' ${IMAGE_ROOTFS}/etc/yum.repos.d/intel-aero.repo
+               Version=${DISTRO_VERSION}
+               Version="${Version%.*}"
```
Comment (anselmolsm) [inline recipes-core/images/intel-aero-image.bb]:
```bash
$ Version="1.5"
$ echo ${Version%.*}
1
```

Try `cut -f  1,2 -d "." # keep fields 1 and 2, separated by dot char`

**Comment by padmashree9110** [comment]
@anselmolsm Thank You, It is done.

**Comment by lucasdemarchi** [comment]
I renamed "Version" to "version" just because I like lowercase better.

It was failing an incremental build, so I added a commit on top to disassociate the package version from the distro version. Updating the package revision when we change a recipe should now always be done since we are supporting packages updates... otherwise users won't get the update if/when we publish the RPMs.

**Comment by lucasdemarchi** [comment]
Applied.

**Pull request closed without merging**